app/frontend/src/components/ApplicationList.vue

```diff
@@ -52,18 +61,23 @@ export default {
   name: 'ApplicationList',
   data: () => ({
     // TEMP for testing skeletons
-    waiting: true
+    waiting: true,
+    clientExistsText: 'Available',
+    clientNotExistsText: 'Not Available'
```

Comment:
Extending on the above comment... we could consider just making this a computed instead and pass in a boolean. Data usually implies it needs to be permuted, but the information stored in those fields are not.

Reply:
I'll see how it shakes out with refactoring to a method. This text is specific to how the component displays the boolean condition from the store's object, so I considered it component-specific data...
But may change after the above